addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CodeInjectionScanRule.java

```diff
@@ -286,6 +287,7 @@ private boolean testAspInjection(String paramName) {
                             .setName(Constant.messages.getString(MESSAGE_PREFIX + "name.asp"))
                             .setParam(paramName)
                             .setAttack(aspPayload)
+                            .setEvidence(PHP_CONTROL_TOKEN)
```

Comment:
Why would the asp alert have php evidence?

Reply:
For no reason, thx. Wil send it with the tests.